code/States/Gamemodes/BaseGamemode.cs

```diff
@@ -24,6 +24,11 @@ public abstract partial class BaseGamemode : BaseState
 	/// </summary>
 	[Net]
 	public TeamManager TeamManager { get; private set; } = null!;
+	/// <summary>
```

Comment:
I know it's annoying but can we ensure there are empty lines between before a doc comment? (i.e. a space between lines 26 and 27 here)

Reply:
And anywhere else too. Not a huge deal but just keep in mind for the future. I wonder if there is an editorconfig option for this.

Reply:
I can sort that in master after it's merged